ouroboros-network-api/src/Ouroboros/Network/PeerSelection/PeerSharing.hs

```diff
@@ -67,14 +67,12 @@ encodeRemoteAddress (SockAddrInet pn w) = CBOR.encodeListLen 3
                            <> CBOR.encodeWord 0
                            <> CBOR.encodeWord32 w
                            <> encodePortNumber pn
-encodeRemoteAddress (SockAddrInet6 pn fi (w1, w2, w3, w4) si) = CBOR.encodeListLen 8
+encodeRemoteAddress (SockAddrInet6 pn _ (w1, w2, w3, w4) _) = CBOR.encodeListLen 6
```

Comment:
It should be: `encodeRemoteAddress :: NodeToNodeVersion -> SockAddr -> CBOR.Encoding`.